smarts/diagnostic/README.md

```diff
@@ -1,22 +0,0 @@
-Performance Diagnostic
```

Comment:
What is the reason for removing this file?

Reply:
This file was moved to the `docs` and can be found at https://smarts.readthedocs.io/en/rtd/resources/diagnostic.html  .

The idea is to concentrate all the documentation at an easy to find centralised place such as the `docs` rather than having them scattered throughout our repository.